js/index.js

```diff
@@ -1 +1,50 @@
 // Your code goes here
+
+const busPic = document.querySelector('.intro img');
+
+busPic.addEventListener('click', (event) => {
+  event.stopPropagation();
+  busPic.setAttribute(
+    'src',
+    'https://thehorse.com/wp-content/uploads/2017/01/iStock-510488648.jpg'
+  );
+});
```

Comment:
If you want to save yourself some headache, you can just write this as:
`busPic.src = 'https://thehorse.com/wp-content/uploads/2017/01/iStock-510488648.jpg'`